### DISCUSSION TITLE
某种程度上这个项目比996.icu更加有意义

### DISCUSSION THREAD

**Discussion by hugetiny** [WeNeedHome/SummaryOfLoanSuspension#1052] [General]
工作毕竟是一种双向选择，我选择这个公司，这公司选择我。没人性的公司太多了，但毕竟可以随便跳槽。
而烂尾楼完全就是地方政府官逼民反了，贷进去款的百姓没啥选择的其实。

**Comment by Accelerator6666** [comment]
说的没错，政府把各种权益都和房地产绑定，逼着别人去买房，但是政府从来不会给你保证或者兜底房子一定能造好。本质上就是政府的责任。
